sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -109,7 +121,13 @@ case class DateSub(startDate: Expression, days: Expression)
     })
   }
 }
-
+/*
+ * Returns the hour componemnt of the string/timestamp/interval
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(timestamp_param) - Returns the hour componemnt of the timestamp value.",
+  extended = "> SELECT _FUNC_('2001-01-02 11:35:55');\n '11'\n" +
```

Comment:
The example should make it clearer that the hour component here is from the 24-hour representation of the timestamp in the local time zone.